joomla/joomla-cms
# [5.2] Installation: Move getOptions() in models to base class

### Summary of Changes
While running phpstan, it threw notices for the ChecksModel and SetupModel versions of getOptions(), since they didn't have proper return statements. Looking at the other models, the code was more complex than necessary. Then also looking at the other models, it is unnecessary to have the same method 4 times in classes inheriting from the same base class, so this PR moves that method to the base class.


### Testing Instructions
Simply run through the installation of Joomla after this change and nothing should be different to before. So realistically this is a matter of codereview.


### Link to documentations
Please select:
